projects/plugins/jetpack/extensions/blocks/button/use-fallback-colors.js

```diff
@@ -0,0 +1,36 @@
+import { useRefEffect } from '@wordpress/compose';
+import { useState } from '@wordpress/element';
+
+export default function useFallbackColors() {
+	const [ fallbacks, setFallbacks ] = useState();
+
+	const ref = useRefEffect( node => {
+		const observer = new MutationObserver( () => {
+			const fallbackBackgroundColor = getComputedStyle( node ).backgroundColor;
+			const fallbackTextColor = getComputedStyle( node ).color;
```

Comment:
I think we can make a small optimization here by avoiding repeated calls to `getComputedStyle( node )`.